### ISSUE TITLE
[Storage] blob upload output is not json but a dynamic progress which  doesn't align with the output format standard

### ISSUE THREAD

**Issue by hasonmsft** [Azure/azure-cli#1566]
Also, it's not aligned with `az storage blob download`
```
C:\tmp>az storage blob upload -c hasonclibugbash -f c:\tmp\local.txt -n upload.txt -o json
Percent complete: %100.0 => progress but not json output
```

**Comment by tjprescott** [comment]
What do you mean it's not aligned with `az storage blob download`?  

The upload command in the SDK does not return a value. If it did, the return value would be in JSON/table/etc. format. The progress indicator uses the `create_blob_from_path`'s progress_callback parameter to provide information to the user via STDERR. In general the CLI is quiet, but in the few cases where informational text is output (such as this), it does not consider the output format.

**Comment by hasonmsft** [comment]
`az storage blob download` doesn't have any progress and will return json output. Could you file a GitHub issue to let Python SDK to return a value to you like other commands: 'az storage container create`? Actaully the upload success should return something like etag, md5, etc...

**Comment by johanste** [comment]
@troydai, can you please open a bug against the SDK. My assumption is that the command will start behaving as requested as soon as the SDK has been updated.

**Comment by troydai** [comment]
Tracking in https://github.com/Azure/azure-storage-python/issues/332. Move to backlog, pending SDK action.

**Comment by troydai** [comment]
SDK has fixed this issue in latest 0.35. Need to update dependency. I'll upgrade storage SDK dep early next sprint.

**Comment by troydai** [comment]
SDK dependency is updated.
